tests/e2e/default/login/renew-token.wdio-spec.js

```diff
@@ -30,7 +30,7 @@ describe('should renew token', async () => {
 });
 
 const getCtxCookieExpiry = async () => {
-  const userCtxCookie = await browserPage.getCookies('userCtx');
+  const userCtxCookie = await chtDbUtils.getCookies('userCtx');
```

Comment:
`.getCookies('userCtx');` the cookies are specific to a browser thing, so we need to take this function out from the `chtDbUtils` and put it in the browser utils. 
What do you think?

Reply:
What about renaming the browser-log.js to browser.js? Then it can have `getCookies` and the logs logic 🤔

Reply:
Actually, @lorerod, we can just use the webdriverio's `browser.getCookies`,  we are already doing this in other places, and we don't have much benefit of having that function in the utils.  Summarizing:
- delete `getCookies` function from `chtDbUtils`
- update this line to use `browser.getCookies('userCtx')` from webdriverio
- and we are good to go :)